app/jobs/bulk_actions_job.rb

```diff
@@ -0,0 +1,48 @@
+class BulkActionsJob < ApplicationJob
+  queue_as :medium
+  attr_accessor :conversations
+
+  def perform(account:, params:, method_type:, user:)
+    @conversation_ids = params.delete(:conversation_ids)
+    @labels = params.delete(:labels)
+    @conversations = account.conversations.where(display_id: @conversation_ids)
+    @method_type = method_type
+    Current.user = user
+    bulk_update(params)
+  end
+
+  def bulk_update(params)
+    if @method_type == :delete
```

Comment:
method type :delete shouldn't be required based on the previous comments since everything related to the action will be wrapped under create